compiler/semantic/proc.go

```diff
@@ -558,41 +558,29 @@ func semProc(ctx context.Context, scope *Scope, p ast.Proc, adaptor proc.DataAda
 			Order: order.Asc, //XXX
 		}, nil
 	case *ast.Over:
-		exprs, err := semExprs(scope, p.Exprs)
+		return semOver(ctx, scope, p, adaptor, head)
+	case *ast.Let:
+		if p.Over == nil {
+			return nil, errors.New("let operator missing traversal in AST")
+		}
+		if p.Over.Scope == nil {
+			return nil, errors.New("let operator missing scope in AST")
+		}
+		scope.Enter()
+		defer scope.Exit()
+		locals, err := semLets(scope, p.Locals)
```

Comment:
Should this be called `semLet` to agree with `ast.Let`?

Reply:
I don't understand...

Reply:
I fully admit that this is marginal territory, but `semLets` feels a little off to me because it's plural while `ast.Let` is singular. But after another look at the code, `semLet` doesn't feel right since it doesn't have an `ast.Let` parameter.

Given the similarity to `semConsts`, how about calling it `semVars`? (They're identical except that the latter calls `scope.DefineVar` instead of `scope.DefineConst`.)